deployments/apiv2/services/users/src/main.py

```diff
@@ -405,14 +413,23 @@ async def _create_user_email(
     con,
     type: str,
     user_id: uuid.UUID,
-    customer_id: uuid.UUID,
+    customer_id: Optional[uuid.UUID],
     scope: List[str],
-    name: str,
+    name: Optional[str],
     email: EmailStr,
 ):
     try:
+        if "user" in scope[0]:
+            account_type = "user"
+            query = "UPDATE users SET pw_reset_verify_link=$1 WHERE id=$2"
+        else:
```

Comment:
this is nitpicky but can you make this so it explicitly checks for each type so that if there's a bug that somehow creates a scope with an `account_type` other than `users` it doesn't allow customer pw resets. Something like

```
if "user" in scope[0]:
    ...
elif "customer" in scope[0]:
    ...
else:
    return an error here
```

This is mostly for being defensive in the code, I don't think it's likely that could happen but if there was a typo in a PR that created say a scope for `users` this would allow the customer password to be changed instead.

Reply:
@jperson yep got it!